public/index.html

```diff
@@ -37,6 +37,7 @@
   <!-- <div id="react"></div> -->
   <div id="registerPanel"></div>
   <div id="instructionDisplay"></div>
+  <div id="miscellaneousPanel"></div>
```

Comment:
can we change this from misc panel to like memoryPanel or something?

also can we change instructionDisplay to instructionPanel so its more consistent? (if you change instructionDisplay make sure to also change the selector in public/assets/styles/style.css)

Reply:
renamed everything to end with Display

Reply:
oh, Ill change everything to end with Panel, actually.